### PR TITLE
HPC : Guide update life cycle policy

### PR DESCRIPTION
**Pull request by pg-ovh**
Update lifecycle policy for HPC VMware on OVHcloud offers.

### CODE REVIEW

File: pages/hosted_private_cloud/hosted_private_cloud_powered_by_vmware/lifecycle_policy/guide.en-gb.md [hunk 1]
```diff
@@ -11,29 +11,29 @@ updated: 2024-01-22
...
 ## Objective
 
-**This page provides an overview of the lifecycle policy of VMware on OVHcloud services.**
+**This page provides an overview of the lifecycle policy of Hosted Private Cloud VMware on OVHcloud services.**
 
-OVHcloud's Hosted Private Cloud VMware service proposes dedicated infrastructure based on VMware SDDC technologies as well as other editors' technologies (such as Veeam or Zerto).
+Hosted Private Cloud VMware on OVHcloud services proposes dedicated infrastructure based on SDDC technologies as well as other editors' technologies (such as Veeam or Zerto).
 
-OVHcloud has a lifecycle policy for the service in order to take into account several factors, such as:
+OVHcloud has a lifecycle policy for the service in order to take into account several factors, such as :
```
Comment (Y0Coss) [inline pages/hosted_private_cloud/hosted_private_cloud_powered_by_vmware/lifecycle_policy/guide.en-gb.md]: There is no space in English before punctuation signs. Please revert this change on lines 18, 21, 22, 129 for all guides except fr-fr and fr-ca.

File: pages/hosted_private_cloud/hosted_private_cloud_powered_by_vmware/lifecycle_policy/guide.en-gb.md [hunk 2]
```diff
@@ -119,18 +119,17 @@ VMware extended the support of NSX Datacenter for vSphere on OVHcloud infrastruc
...
-To know the lifecycle policy of Zerto products, please refer to the editor's publication accessible at the following URL:
+To know the lifecycle policy of Zerto products, please refer to the editor's publication accessible at the following URL :
 
-[Zerto Virtual Replication Product Version Lifecycle Matrix](https://help.zerto.com/bundle/Lifecycle.Matrix.HTML/page/Content/Lifecycle_Matrix/Lifecycle_Matrix.htm#zerto_virtual_replication_product_version_lifecycle_matrix_r_893035264_1010900)
+- [Zerto Virtual Replication Product Version Lifecycle Matrix](https://help.zerto.com/bundle/Lifecycle.Matrix.HTML/page/Content/Lifecycle_Matrix/Lifecycle_Matrix.htm#zerto_virtual_replication_product_version_lifecycle_matrix_r_893035264_1010900)
 
 ## Go further
 
-Join our community of users on <https://community.ovh.com/en/>.
+Join our [community of users](/links/manager).
```
Comment (Y0Coss) [inline pages/hosted_private_cloud/hosted_private_cloud_powered_by_vmware/lifecycle_policy/guide.en-gb.md]: wrong link on all versions except fr-fr and fr-ca
It should be `(/links/community)`

**Review by Y0Coss — CHANGES_REQUESTED** [review]

**Review by Y0Coss — APPROVED** [review]

**Comment by Y0Coss** [comment]
Waiting for the review from @franckcosta to merge the PR.
**Note**: I'd recommend switching the dates format to the international format YYYY-MM-DD

**Comment by pg-ovh** [comment]
Thanks @Y0Coss it's all good with @franckcosta.

**Comment by Y0Coss** [comment]
Thank you @pg-ovh for your contribution

**Pull request merged**